package.json

```diff
@@ -16,11 +16,11 @@
   },
   "dependencies": {
     "ast-types": "^0.8.12",
+    "babel": "^5.8.23",
```

Comment:
@thejameskyle I'd like to enable all transformations, up to stage 0 - but it looks like [they are defined in babel, not babylon](https://github.com/babel/babel/blob/master/packages/babel/src/transformation/index.js) - is there a way to trigger these with babylon alone?